docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -1,7 +1,186 @@
 # What does `security-status` do?
 
+## Command output
+
 The `security-status` command is used to get an overview of the packages
-installed on your machine.
+installed on your machine, and an overview of the security coverage for those
+packages.
+
+If you run the `pro security-status` command, the first blocks of information
+you see look like:
+
+```
+2871 packages installed:
+     2337 packages from Ubuntu Main/Restricted repository
+     504 packages from Ubuntu Universe/Multiverse repository
+     8 packages from third parties
+     22 packages no longer available for download
+
+To get more information about the packages, run
+    pro security-status --help
+for a list of available options.
+```
+
+Those are counts for the `apt` packages installed in the system, sorted
+between the packages in main, universe, third party packages, and packages
+no longer available. You will also see a hint to run
+`pro security-status --help` to get more information.
+
```

Comment:
In general, before each different subsection, I'd put a subheading to aid navigation. E.g. the first one on this line, before "To get this package information...". Each of these code snippets + description seems to cover a distinct/new thing so it would help to signpost them with headers.